README.md

```diff
@@ -37,8 +37,9 @@ or clone the repo for the most up-to-date features
 ```bash
 git clone https://github.com/strakam/generals-bots
 cd generals-bots
-pip install -e .
+make install
 ```
+Under the hood, `make install` installs [poetry](https://python-poetry.org/) and the package using poetry.
```

Comment:
nitpick: Make this line a part of `> [!Note]`, see [here](https://github.com/strakam/generals-bots/blob/f157ca7d4af6cc3423bd0d72a381652f02944056/README.md?plain=1#L27)